src/templates/pt/base.html

```diff
@@ -109,15 +109,15 @@
 {% endif %}
 
 {% block contributors_description %}
-O Web Almanac Isso foi possível graças ao trabalho árduo da comunidade da web. {{ self.contributors() }} as pessoas ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção.
+O Web Almanac Isso foi possível graças ao trabalho árduo da comunidade da web. {{ self.contributors() }} as pessoas ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção do Web Almanac {{ year }}.
```

Comment:
You could change that sentence to "O Web Almanac só foi possível graças ao trabalho árduo da comunidade web. {{ self.contributors() }} pessoas ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção do Web Almanac {{ year }}."

Reply:
Sounds good. Adding as suggestion:

```suggestion
O Web Almanac só foi possível graças ao trabalho árduo da comunidade web. {{ self.contributors() }} pessoas ofereceram inúmeras horas nas fases de planejamento, pesquisa, redação e produção do Web Almanac {{ year }}.
```